pyoffers/models/affiliate.py

```diff
@@ -1,8 +1,19 @@
 # coding: utf-8
-from .core import Model, ModelManager
+from .core import InvisibleModelManager, Model, ModelManager
 from .offer_file import OfferFileManager
 
 
+class AffiliateOffer(Model):
+    """"
+    Model that defines relation between Offer and Affiliate
+    """
+    pass
```

Comment:
`pass` is redundant

Reply:
oops, `ValuesDict` shouldn't even be here, we have removed it in previous pull request. This is a merge mistake.